service/src/test/java/uk/gov/hmcts/reform/fpl/validators/HasAttachedDocumentValidatorTest.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.fpl.validators;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.fpl.interfaces.UploadDocumentsGroup;
+import uk.gov.hmcts.reform.fpl.model.common.Document;
+import uk.gov.hmcts.reform.fpl.service.ValidateGroupService;
+
+import java.util.List;
+import javax.validation.Validation;
+import javax.validation.Validator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.utils.CaseDataGeneratorHelper.createDocumentWithAttachedFile;
+
+@ExtendWith(SpringExtension.class)
+public class HasAttachedDocumentValidatorTest {
```

Comment:
I believe we should test happy path scenario as well here - i.e. no errors returned what do you think?